### ISSUE TITLE
More info in readme

### ISSUE THREAD

**Issue by oxinabox** [bramtayl/Remotes.jl#1]
It would be nice to have more info in the readme.
To explain what this is for.

**Comment by bramtayl** [comment]
Sure, absolutely. Also, I need to change the name of the package to me more descriptive, do you have any suggestions?

**Comment by oxinabox** [comment]
i'm honestly not sure what this package does. I have a few hopes.

**Comment by bramtayl** [comment]
If you've tried out `PackageGenerator.jl` is basically that but only the online portion. I made it because `PkgTemplates.jl` seems to be much better/more widely used but is missing any online functionality.

**Comment by oxinabox** [comment]
I think that is why I only tried PkgTemplates once, then stopped using it.

What I really want is PackageGenerator.jl, plus a button to go an create the GitHub, repo and enable Travis and AppVeyor (via their APIs, or via using say Selenium to drive a web-browser),

**Comment by bramtayl** [comment]
Yup it does all that. No selenium required all three have REST apis.

**Comment by bramtayl** [comment]
This package is now published at https://github.com/bramtayl/OnlinePackage.jl

**Comment by bramtayl** [comment]
I've updated the readme, but let me know if you think there's more I should add.

**Comment by oxinabox** [comment]
You know you can rename github repos without any issues what so ever?
Github creates all the redirects automatically

**Comment by bramtayl** [comment]
Good to know!
